src/routes/+layout.server.ts

```diff
@@ -20,6 +21,13 @@ export async function load(event) {
 				return `width=${runtimeConfig.viewportContentWidth}`;
 		}
 	})();
+	let bolt12Address = '';
+
+	try {
+		bolt12Address = await phoenixdGetBolt12();
+	} catch (error) {
+		bolt12Address = 'Bolt12';
+	}
```

Comment:
This does an API call to phoenixd on the backend for every single page render. It doesn't even check that phoenixd is configured (with `isPhoenixdConfigured`)

This will add latency to the app, especially if phoenixd has troubles.

Please call it only when necessary. Maybe it's there because of CMS page, but then you can call it when renderning CMS only if the bolt12 qr code is asked for (and only if the QR Code CMS Element is rendered)